### DISCUSSION TITLE
线上会议记录 2024 9-10 月

### DISCUSSION THREAD

**Discussion by jiacai2050** [zigcc/forum#142] [General]
参会方式见：https://ziglang.cc/post/news/

这里记录一下每次的大致内容。

**Comment by jiacai2050** [comment]
# 2024-08-31

@jiacai2050 
- https://github.com/allyourcodebase/lz4
- https://zigcli.liujiacai.net/programs/tcp-proxy/

**Comment by jinzhongjia** [comment]
@jinzhongjia
unixsocket 的可打断io
zig neovim 插件的支持

**Comment by flyfish30** [comment]
zig的函数式编程基础库，现在已经有functor, applicative,monad的完整支持。
